packages/connect/package.json

```diff
@@ -46,7 +47,7 @@
     "@types/chai-spies": "1.0.3",
     "@types/debug": "4.1.7",
     "@types/mocha": "9.0.0",
-    "@types/node": "16.11.18",
+    "@types/node": "16.11.19",
```

Comment:
necessary to use AbortController in TCP socket, see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57919